pysoa/server/standalone.py

```diff
@@ -19,14 +22,63 @@
     exit(99)
 
 
+def _get_arg_parser():
+    import argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-f', '--fork-processes', '--fork',
+        help='The number of processes to fork (if 0, 1, or none, no process is forked; the server is run directly)',
+        required=False,
+        type=int,
+        default=0,
+    )
+    return parser
+
+
+def _get_args(parser):
+    return parser.parse_known_args()[0]
+
+
+def _run_server(args, server_class):
+    if args.fork_processes > 1:
```

Comment:
Do you want to have a max cap? maybe `(multiprocessing.cpu_count() or 1) * 10`

Reply:
That's not a bad idea. I'm actually going to set it a bit lower than that—5 processes per CPU. Thanks!